jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/LocalBaseImageSteps.java

```diff
@@ -90,25 +91,58 @@ static boolean isGzipped(Path path) throws IOException {
 
   private final ExecutorService executorService;
   private final BuildConfiguration buildConfiguration;
-  private final Path tarPath;
   private final ProgressEventDispatcher.Factory progressEventDispatcherFactory;
-  private final TempDirectoryProvider tempDirectoryProvider;
 
-  ExtractTarStep(
+  LocalBaseImageSteps(
       ExecutorService executorService,
       BuildConfiguration buildConfiguration,
-      Path tarPath,
-      ProgressEventDispatcher.Factory progressEventDispatcherFactory,
-      TempDirectoryProvider tempDirectoryProvider) {
+      ProgressEventDispatcher.Factory progressEventDispatcherFactory) {
     this.executorService = executorService;
     this.buildConfiguration = buildConfiguration;
-    this.tarPath = tarPath;
     this.progressEventDispatcherFactory = progressEventDispatcherFactory;
-    this.tempDirectoryProvider = tempDirectoryProvider;
   }
 
-  @Override
-  public LocalImage call()
+  Callable<LocalImage> processDockerDaemonBaseImageStep(DockerClient dockerClient) {
```

Comment:
`loadDockerDaemonBaseImage`?  Or `loadImageFromDockerDaemonStep`?

Reply:
I think we should avoid "`load`" so we don't confuse it with `docker load`, would `saveDockerDaemonBaseImageStep` be better?